rktl_sim/src/simulator/sim.py

```diff
@@ -74,20 +89,37 @@ def __init__(self, props, urdf_paths, spawn_bounds, render_enabled):
 
         if 'engine' in self.props and self.props['engine'] is not None:
             p.setPhysicsEngineParameter(**self.props['engine'])
+        # TODO: why gravity not accurate?
         p.setGravity(0, 0, -10)
 
     def configure_dynamics(self, body_id, body_type):
+        """
+        Configure the dynamics of the car.
```

Comment:
Need to be really careful about comments that restate function or variable names. Also strive to make comments concise, yet elucidating.